backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java

```diff
@@ -1870,7 +1870,13 @@ private Boolean blobTypeExistsInList(List<BlobType> blobTypeList, Blob.ContentCa
       boolean partNumberSpecified,
       S3KeyFunction initializeMultipart)
       throws ModelDBException {
-    UploadStatusEntity uploadStatusEntity = getUploadStatusEntity(session, computeSha, contentCase);
+    List<UploadStatusEntity> uploadStatusEntities =
+        getUploadStatusEntity(session, computeSha, contentCase);
+    UploadStatusEntity uploadStatusEntity = null;
+    if (uploadStatusEntities != null && !uploadStatusEntities.isEmpty()) {
+      uploadStatusEntity = uploadStatusEntities.get(0);
```

Comment:
we are silently ignoring other items 
a warn log if there are multiple uploadStatusEntities returned will be helpful for debugging

Reply:
added